LIMS_IMAGE/web/src/urls.py

```diff
@@ -16,13 +16,19 @@
 from django.contrib import admin
 from django.urls import path, include
 from django.views.generic.base import TemplateView
+from django.conf import settings
+from django.conf.urls.static import static
 
 # Django reads URL patterns from top to bottom
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('accounts/', include('accounts.urls')),
+    # TODO: why are there two accounts urls?
     path('accounts/', include('django.contrib.auth.urls')),
     path('orders/', include('orders.urls')),
     path('laboratory/', include('laboratory.urls')),
     path('', TemplateView.as_view(template_name='home.html'), name='home'),  # new
```

Comment:
When navigating directly to the site root (`locahost:8000`) in the browser search bar when logged in as a user, the site does not redirect the user to their appropriate homepage. 

**Expected:**
![image](https://user-images.githubusercontent.com/39775792/154772637-744c9e06-bf75-4022-a9d4-851039448ef0.png)

**Actual:**
![image](https://user-images.githubusercontent.com/39775792/154772734-9152a471-4860-4d80-8da4-c19db5043e2a.png)

The fix is to create an app for the base homepage instead of using the built in `TemplateView.as_view()` function, similar to the other URL patterns. Then a custom view can be made to redirect a user correctly with the logic we've already implemented for the other views.
![image](https://user-images.githubusercontent.com/39775792/154772827-e9870f31-1730-4620-aa84-c8f943acd45f.png)

Reply:
https://github.com/shepkeira/LIMS/pull/248 <- this should fix it